### PR TITLE
fix: get toolchains for remote builds working correctly

### PR DESCRIPTION
**Pull request by justbuchanan**
Maybe this will fix the buildbot failures? https://buildbot-dot-kythe-repo.appspot.com/#/builders/20/builds/50

### CODE REVIEW

**Comment by justbuchanan** [comment]
Looks like the problem is that the `bazel_toolchains` dep is not pulled into the lang-proto workspace. The main kythe repo has this in the WORKSPACE file:

```python
http_archive(
    name = "bazel_toolchains",
    sha256 = "0ffaab86bed3a0c8463dd63b1fe2218d8cad09e7f877075bf028f202f8df1ddc",
    strip_prefix = "bazel-toolchains-5ce127aee3b4c22ab76071de972b71190f29be6e",
    urls = [
        "https://mirror.bazel.build/github.com/bazelbuild/bazel-toolchains/archive/5ce127aee3b4c22ab76071de972b71190f29be6e.tar.gz",
        "https://github.com/bazelbuild/bazel-toolchains/archive/5ce127aee3b4c22ab76071de972b71190f29be6e.tar.gz",
    ],
)
```

@schroederc, would it be best to add this to the WORKSPACE directly or to change the kythe repo so this gets added the via `kythe_dependencies()` macro?

**Comment by schroederc** [comment]
Adding it here seems best since it is only referenced by the `.bazelrc` file.

File: WORKSPACE [hunk 1]
```diff
@@ -2,10 +2,20 @@ load("@bazel_tools//tools/build_defs/repo:git.bzl", "git_repository")
 
 git_repository(
     name = "io_kythe",
-    commit = "01617d9cc0753550964c57af57c64644a0808798",
+    commit = "04f07ead07a2d6073f9c59e846117f096e45e91e",
     remote = "https://github.com/kythe/kythe",
 )
 
+http_archive(
```
Comment (schroederc) [inline WORKSPACE]: Needs `load("@bazel_tools//tools/build_defs/repo:http.bzl", "http_archive")`

Comment (schroederc) [reply WORKSPACE]: `git_archive` is not defined in the same file.

**Comment by justbuchanan** [comment]
It works! We're down to just the one test failure we had before.

**Pull request merged**